test/evergreen.yml

```diff
@@ -358,6 +358,17 @@ functions:
           cd build_posix
           ${test_env_vars|} ${make_command|make} VERBOSE=1 check ${smp_command|} 2>&1
         fi
+
+  "cppsuite test":
+   - command: shell.exec
+     params:
+       working_dir: "wiredtiger/build_posix/"
+       script: |
+         set -o errexit
+         set -o verbose
+
+         ${test_env_vars|} $(pwd)/test/cppsuite/run -t ${test_name} -C '${test_config}' -f test/cppsuite/configs/${test_config_filename} -l 2
```

Comment:
I feel embedding `test/cppsuite/configs/` into `${test_config_filename}` will be more readable and flexible (in case configs can be saved to different folders in the future)?

Reply:
That's a good point, fixed in 42c61cf43b868a5a860a11136520e8ae58536864